use-cases.html

```diff
@@ -0,0 +1,303 @@
+          <p>
+            Someone visiting a foreign city could access a web application that
+            allows users to search or browse through a database of tourist
+            attractions. Using the Geolocation Sensor API, the web application has access to
+            the user's approximate current position and is therefore able to rank the
+            search results by proximity to the user's location.
+          </p>
+        </section>
+
+      </section>
+
+      <section id="usecases_continuous_foreground">
+        <h3>Foreground—Continuous geolocation updates</h3>
+
+        <section id="usecase_local_info">
+          <h4>Up-to-date local information</h4>
+
+          <p>
+            A widget-like web application that shows the weather or news that are
+            relevant to the user's current area can use the Geolocation Sensor API to
+            register for location updates. If the user's position changes, the widget
+            can adapt the content accordingly.
+          </p>
+        </section>
+
+        <section id="usecase_poi_alert">
+          <h4>Alerts when points of interest are in the user's vicinity</h4>
+
+          <p>
+            A tour guide web application can use the Geolocation Sensor API to monitor the
+            user's position and trigger visual or audio notifications when interesting
+            places are in the vicinity. An online task management system can trigger
+            reminders when the user is in the proximity of landmarks that are
+            associated with certain tasks.
+          </p>
+        </section>
+
+        <section id="usecase_show_position">
+          <h4>Show the user's position on a map</h4>
+
+          <p>
+            A user finds themselves in an unfamiliar area. They want to check their
+            position so they use their handheld device to navigate to a web-based
+            mapping application that can pinpoint their exact location on the map
+            using the Geolocation Sensor API. They then ask the web application to provide
+            driving directions from their current position to their desired destination,
+            essentially responding to the question "Where am I going?".
+          </p>
+        </section>
+
+      </section>
+
+      <section id="usecases_continuous_background">
+        <h3>Background—Continuous geolocation updates</h3>
+
+        <section id="usecase_social_networking">
+          <h4>Location-tagged status updates in social networking applications</h4>
+
+          <p>
+            A social networking application allows its users to automatically tag their
+            status updates with location information. It does this by monitoring the
+            user's position with the Geolocation Sensor API. Each user can control the
+            granularity of the location information (e.g., city or neighborhood level)
+            that is shared with the other users. Any user can also see their network
```

Comment:
This seems to be conflating two use cases, a) geotagging status updates and b) social sharing of real-time location. The former only requires foreground-one-off location while the latter is what requires background-continuous location.

Reply:
I think this is really by design conflated. You check where your friends are (and see where you are) in the foreground, but indeed of course your location gets shared in the background, too.

Reply:
Can you make it a little clearer that we are intentionally conflating the two? "Location-tagged status updates" are a feature multiple social media platforms already support that doesn't use background location. We should make it clear that this is different.

Reply:
I will highlight this aspect.